### ISSUE TITLE
Makerbeam/kit V0 extrusion compatability

### ISSUE THREAD

**Issue by robertsamples** [JackJack3231/V0-Auxiliary-Fan#2]
Might be worth considering making the tab that slots into the extrusions more narrow or at least adding a note in documentation. I think the current tab is too thick and is only compatible with wider slot extrusions, a lot of kits use makerbeam-xl style extrusions with a 3.0-3.2 mm slots. Its easy to change in CAD but at least adding a note in documentation about this would be nice.

**Comment by JackJack3231** [comment]
The tab is 3.2mm wide (in CAD) to make it a snug fit and the whole thing more solid.
I tried making thinner versions in the past and it made it very loose and more or less pointless (at least on my MakerBeamXL Frame).
The idea is if it is too tight people can sand down the tab a bit to make it slot in easier, however I forgot to add a comment about that in the description/assembly instructions. I added those with f164e16a5d0a651776174ad886a23f61d30a0b10.

I might redesign the tab to give a bit of flex and make it easier to insert for frames with quite narrow slots.
